constraint-solver/src/grouped_expression.rs

```diff
@@ -73,10 +75,9 @@ pub struct QuadraticSymbolicExpressionImpl<T, V> {
 
 // TODO: This type is equivalent to a pre-refactoring version of `QuadraticSymbolicExpressionImpl`.
```

Comment:
is this TODO still relevant/correct?

Reply:
Yes, I will do this in a step by step way one PR at a time and at the end the type will be removed.